services/web/server/tests/unit/with_dbs/01/groups/test_groups_handlers_crud.py

```diff
@@ -85,15 +85,23 @@ async def test_list_user_groups_and_try_modify_organizations(
         key=by_gid,
     ) == sorted(standard_groups, key=by_gid)
 
-    for group in standard_groups:
+    for i, group in enumerate(standard_groups):
```

Comment:
Minor: A former colleague introduced to me the very nice convention of always using `ii` or `jj` for counters in loops like this. The reasoning is that often there is an integer passed as an arguments (his reasoning was concerned with C code) if you do write algorithmic code. And then one often reaches conflicts, so it is nice to have "special" conventions for the indices in loops. I guess this is not as needed in python code as in C code, but I like the convention very much because it is clear where a counter comes from.

Reply:
Will apply this conversion in a separate PR i am preparing with minor structural changes.